addons/web_unsplash/static/src/scss/unsplash.scss

```diff
@@ -1,11 +1,19 @@
 .unsplash_error {
-    padding: 30px 0;
-    .access_key_box {
-        padding: 9px;
-        background-color: #fcfcfc;
-        border: 1px solid #ededee;
-        input {
-            min-width: 300px;
+    .alert-info {
+        *:not(h4):not(p) {
+            display: inline;
+        }
+        .o_unsplash_setup_key, .o_unsplash_setup_id {
+            @include media-breakpoint-down(lg) {
+                display: block;
+                margin: 15px 0;
+            }
+        }
+    }
+    .save_unsplash {
+        margin: 0 0 3px 15px;
+        @include media-breakpoint-down(lg) {
+            margin: 0;
         }
     }
 }
```

Comment:
Let's make this better, we don't need any css to design the mockup I saw. Remove this css file entirely and use a nice bootstrap XML structure to make this alert message.